### PR TITLE
Feature/basic components

### PR DESCRIPTION
**Pull request by farinalvaro**
Solving #9 UI Components for Listing/Filtering.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +27,8 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.27",
     "@fortawesome/free-solid-svg-icons": "^5.12.1",
     "@fortawesome/react-fontawesome": "^0.1.9",
+    "@material-ui/core": "^4.9.7",
+    "@material-ui/icons": "^4.9.1",
     "bootstrap": "^4.3.1",
```
Comment (jurajpiar) [inline package.json]: Did we not want to remove bootstrap dependency?

Comment (farinalvaro) [reply package.json]: @jurajpiar you are right! I just removed it.

File: src/rifui/components/molecules/RangeSliderWithInputs.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+}
+
+const RangeSliderWithInputs: FC<IRangeSliderWithInputsProps> = ({ values, units, ...rest }) => {
+  const classes = useStyles();
+
+  const maxValue = rest.max || values.end;
+  const minValue = rest.min || values.start;
+  const step = rest.step || 1;
+
+  const [state, setState] = useState<ISliderData>({
```
Comment (jurajpiar) [inline src/rifui/components/molecules/RangeSliderWithInputs.tsx]: I'd call it something more concrete

File: src/rifui/components/molecules/RangeSliderWithInputs.tsx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        ...state,
+        endValue: state.startValue
+      });
+    } else if (state.endValue > maxValue) {
+      setState({
+        ...state,
+        endValue: maxValue
+      });
+    }
+  };
```
Comment (jurajpiar) [inline src/rifui/components/molecules/RangeSliderWithInputs.tsx]: splitting the state should also simplify these handlers

File: src/rifui/components/molecules/RangeSliderWithInputs.tsx (hunk 2 above)
Comment (farinalvaro) [reply src/rifui/components/molecules/RangeSliderWithInputs.tsx]: Solved by splitting the state as suggested, thanks!

File: src/rifui/components/molecules/RangeSliderWithInputs.tsx (hunk 3 above)
Comment (farinalvaro) [reply src/rifui/components/molecules/RangeSliderWithInputs.tsx]: Splitted , thanks for the suggestion!

File: package.json [hunk 4]
```diff
@@ -27,7 +27,8 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.27",
     "@fortawesome/free-solid-svg-icons": "^5.12.1",
     "@fortawesome/react-fontawesome": "^0.1.9",
-    "bootstrap": "^4.3.1",
+    "@material-ui/core": "^4.9.7",
+    "@material-ui/icons": "^4.9.1",
     "formik": "^2.1.4",
     "react": "^16.13.0",
     "react-bootstrap": "^1.0.0-beta.12",
```
Comment (vojtechsimetka) [inline package.json]: this should also be removed

File: src/rifui/index.ts [hunk 5]
```diff
@@ -1,8 +1,5 @@
 // CSS files
 import './css/style.css';
```
Comment (vojtechsimetka) [inline src/rifui/index.ts]: you can probably drop this as well, it was for the old library

File: package.json (hunk 4 above)
Comment (farinalvaro) [reply package.json]: Solving as we agreed to remove react-bootstrap as a new task and create another PR :)

File: src/rifui/index.ts (hunk 5 above)
Comment (farinalvaro) [reply src/rifui/index.ts]: Will drop it when we have new styles

**Comment by farinalvaro** [comment]
Finished the components for filtering/listing

**Pull request merged**